ee/modules/110-istio/openapi/config-values.yaml

```diff
@@ -138,6 +138,14 @@ properties:
         description: Turn on or off tracing collection and displaying in kiali.
         default: false
         x-examples: [true]
+      sampling:
+        type: number
+        minimum: 0.01
+        maximum: 100.0
```

Comment:
From this description, it is hard to tell units. Percents?

Reply:
Yes, it's a percentage. Add units to the description of config-values?